ThunderCloud.xcodeproj/project.pbxproj

```diff
@@ -1860,7 +1860,7 @@
 				IPHONEOS_DEPLOYMENT_TARGET = 11.0;
 				LD_RUNPATH_SEARCH_PATHS = "$(inherited) @executable_path/Frameworks @loader_path/Frameworks";
 				LIBRARY_SEARCH_PATHS = "$(inherited)";
-				MARKETING_VERSION = 3.0.2;
+				MARKETING_VERSION = 3.0.3;
```

Comment:
`3.0.3` is already out? Is this just from back-merging into your branch?

Reply:
Yep back merging :)